Darwin-arm64.cmake

```diff
@@ -0,0 +1,199 @@
+# test args
+
+set(LLVM_LIT_ARGS "--xunit-xml-output=testresults.xunit.xml -v" CACHE STRING "")
```

Comment:
I'm slightly torn on this - I've tried to avoid having paths listed in the cache as those are for the user to specify.